src/psyclone/psyir/frontend/fparser2.py

```diff
@@ -72,6 +72,10 @@
     Fortran2003.Char_Literal_Constant: ScalarType.Intrinsic.CHARACTER,
     Fortran2003.Int_Literal_Constant: ScalarType.Intrinsic.INTEGER}
 
+# Mapping from Fortran intent to PSyIR access type
+INTENT_MAPPING = {"in": ArgumentInterface.Access.READ,
+                  "out": ArgumentInterface.Access.WRITE,
+                  "inout": ArgumentInterface.Access.READWRITE}
 
```

Comment:
pycodestyle says it was expecting 2 blank lines here.

Reply:
Oops.